iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/DanglingLineAdapter.java

```diff
@@ -92,12 +92,8 @@ public String getUcteXnodeCode() {
     }
 
     @Override
+    @Deprecated
     public CurrentLimits getCurrentLimits() {
         return getDelegate().getCurrentLimits();
     }
-
-    @Override
-    public CurrentLimitsAdder newCurrentLimits() {
-        return getDelegate().newCurrentLimits();
-    }
```

Comment:
Why this method is removed?

Reply:
I deplaced it in `AbstractInjectionAdapter`